specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2020-06-01-preview/examples/ApiManagementUndelete.json

```diff
@@ -0,0 +1,90 @@
+{
+  "parameters": {
```

Comment:
You shouldn't be restoring with a PATCH API.  PATCH should only be valid on resources that exist.  The service you are restoring does not exist according to ARM when it is in a soft delete state.  Please use PUT for this.  "location" will also be required in the request and must match the location of the original service to route correctly

Reply:
oops, wrongly placed an example link